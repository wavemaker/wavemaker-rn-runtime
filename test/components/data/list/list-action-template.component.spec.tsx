import React from 'react';
import renderer from 'react-test-renderer';
import WmListActionTemplate from '@wavemaker/app-rn-runtime/components/data/list/list-action-template/list-action-template.component';
describe('Test ListActionTemplate component', () => {
    test('Check validity of sample component', () => {
      const tree = renderer.create(<WmListActionTemplate name="test_ListActionTemplate"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});rr