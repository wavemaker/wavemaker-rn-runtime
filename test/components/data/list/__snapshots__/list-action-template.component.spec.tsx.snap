// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ListActionTemplate component Check validity of sample component 1`] = `
<View
  onLayout={[Function]}
  style={
    {
      "height": 0,
      "width": 0,
    }
  }
/>
`;
